agent/doc/swdesign/plantuml/seq_restart_workload_on_create_failure.plantuml

```diff
@@ -0,0 +1,30 @@
+@startuml seq_restart_workload_on_create_failure
+!include ../../../../resources/doc/plantuml/clean.skin
+
+box "Ankaios Agent"
+    participant "RuntimeFacade" as runtime_facade
+    participant "WorkloadCommandChannel" as workload_command
+    participant "WorkloadControlLoop" as workload_control_loop
+
+    participant "Specific Runtime Connector\n e.g., PodmanRuntime" as runtime
+end box
+
+activate runtime_facade
+runtime_facade ->> runtime ++: create workload
+runtime -->> runtime_facade--
+opt create workload fails
+    runtime_facade ->> workload_command ++: restart
+    runtime_facade ->> workload_control_loop ++: listen for commands
+    workload_command ->> workload_control_loop ++: restart
```

Comment:
Maybe we can show this the other way around as the control loop is reading from the channel.
Let's have a chat about the diagrams when you have some time.

Reply:
Same for this, I have updated and commit both diagrams for restart on create and restart on update. Please review them again.